lsp-mode.el

```diff
@@ -446,15 +450,42 @@ must be used for handling a particular message.")
   "Show MSG in eldoc."
   (run-with-idle-timer 0 nil (lambda () (eldoc-message msg))))
 
-(defun lsp-message (format &rest args)
-  "Wrapper over `message' which preserves the `eldoc-message'.
-FORMAT with ARGS are the original message formats."
-  (let ((inhibit-message lsp-inhibit-message)
-        (last eldoc-last-message))
-    (apply #'message format args)
-
-    (when lsp-inhibit-message
-      (lsp--eldoc-message last))))
+(defun lsp-log (format &rest args)
```

Comment:
can you defalias `lsp-message`? -  it is used from the other packages like lsp-java (it was public)?

Reply:
Maybe use `make-obsolete-variable` instead of `defalias`?